ivy/functional/frontends/torch/tensor.py

```diff
@@ -100,9 +100,20 @@ def requires_grad(self):
     def is_leaf(self):
         return self._is_leaf
 
+    @property
+    def get_device(self):
+        if self.device == "cpu":
+            return -1
+        else:
+            return int(self.devices.split(':')[-1])
+
     # Setters #
     # --------#
 
+    @device.setter
+    def cuda(self):
```

Comment:
Can you conform it to the original torch method [here](https://pytorch.org/docs/stable/generated/torch.Tensor.cuda.html). We should able to mention the device at-least.

Reply:
@Dsantra92 can you check it again

Reply:
@Dsantra92 hello?